apps/website/src/pages/docs/overlay.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Overlay
+description: Overlays hide content from the user with a blur effect and loading indicator.
+layout: ./_layout.astro
+propsPath: '@itwin/itwinui-react/esm/core/Overlay/Overlay.d.ts'
+thumbnail: Overlay
+group: inputs
```

Comment:
Overlay is not in inputs. Move it to core.

Reply:
done :)

Reply:
done